moped-database/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
 
 
     moped-pgsql:
-        image: "frankinaustin/postgis:16-3.5"
+        image: "atddocker/postgis:unified-db"
```

Comment:
at first, i was questioning not having the versions in the tag, but then I reminded myself that they have no real connection to the versions in the image and just as well be `latest-db-version`. So, I like the idea of not needing to PR in these changes every update.

I am curious for other thoughts too. It seems straightforward enough to check the local DB for the postgres and postgis versions but maybe I am missing something.